serviceAccessConfig/lib/serviceAccessConfig/apache.py

```diff
@@ -0,0 +1,147 @@
+#  Copyright (C) 2016 SUSE LLC, Robert Schweikert <rjschwei@suse.com>
+#  All rights reserved.
+#
+#  This file is part of serviceAccessConfig
+#
+#  This program is free software: you can redistribute it and/or modify
+#  it under the terms of the GNU General Public License as published by
+#  the Free Software Foundation, either version 3 of the License, or
+#  (at your option) any later version.
+#
+#  This program is distributed in the hope that it will be useful,
+#  but WITHOUT ANY WARRANTY; without even the implied warranty of
+#  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+#  GNU General Public License for more details.
+#
+#  You should have received a copy of the GNU General Public License
+#  along with this program.  If not, see <http://www.gnu.org/licenses/>.
+#
+
+"""serviceAccessConfig plugin for Apache"""
+
+import ConfigParser
+import glob
+import logging
+import os
+import re
+
+from accessrulegenerator import ServiceAccessGenerator
+from generatorexceptions import *
+
+apache_directory_directive = re.compile(r'\s*<Directory .*>')
```

Comment:
Because I want the expression to be compiled only once and I did not want to make it a member of the class.